src/Native/System.Security.Cryptography.Native/openssl.c

```diff
@@ -76,11 +78,11 @@ Return values:
 1: Data was copied
 Any negative value: The input buffer size was reported as insufficient. A buffer of size ABS(return) is required.
 */
-int
+int32_t
 GetX509Thumbprint(
     X509* x509,
-    unsigned char* pBuf,
-    int cBuf)
+    uint8_t* pBuf,
```

Comment:
> We decided to change this for strings? 

See the conversation here: https://github.com/dotnet/corefx/pull/3868/files#r42076470
For plain buffers, use uint8_t*, but keep char* for actual strings.  GetX509Thumbprint uses a byte[].  I left the normal strings using char*.